openpype/hosts/fusion/plugins/publish/extract_render_local.py

```diff
@@ -71,11 +79,11 @@ def render_once(self, context):
 
         savers_to_render = [
             # Get the saver tool from the instance
-            instance[0] for instance in context if
+            instance.data["tool"] for instance in context if
             # Only active instances
             instance.data.get("publish", True) and
             # Only render.local instances
-            "render.local" in instance.data["families"]
+            "render.local" in instance.data.get("families")
```

Comment:
```suggestion
            "render.local" in instance.data.get("families", [])
```
Without this fallback the `.get` doesn't make much sense since the `in` check doesn't work against `None`. So either add the fallback, or keep the old code behavior.